docs/howto/manage-configurations.md

```diff
@@ -30,21 +49,28 @@ In the `src/charm.py` file of the charm project, in the `__init__` function of t
 self.framework.observe(self.on.config_changed, self._on_config_changed)
 ```
 
-Then, in the body of the charm definition, define the event handler. Here you may want to read the current configuration value, validate it (Juju only checks that the *type* is valid), and log it, among other things. Sample code for an option called `server-port`, with type `int`, and default value `8000`:
+Also in the `__init__` function, load the config into the config class that you
+defined. Pass `errors='blocked'` to have the charm exit after setting a blocked
+status if the configuration doesn't validate against the class you defined.
+
+```python
+self.typed_config = self.load_config(WikiConfig, errors='blocked')
+```
 
- ```python
-def _on_config_changed(self, event):
-    port = self.config["server-port"] 
+Then, in the body of the charm definition, define the event handler.
 
-    if port == 22:
-        self.unit.status = ops.BlockedStatus("invalid port number, 22 is reserved for SSH")
+```python
+def _on_config_changed(self, event: ops.ConfigChangedEvent):
+    name = self.typed_config.name
+    existing_name = self.get_wiki_name()
+    if name == existing_name:
+        # Nothing to do.
         return
-    
-    logger.debug("New application port is requested: %s", port)
-    self._update_layer_and_restart(None)
+    logger.info('Changing wiki name to %s', name)
+    self.set_wiki_name(name)
 ```
 
-> See more: [](ops.CharmBase.config)
+> See more: [](ops.CharmBase.load_config)
```

Comment:
Do we want to link both `.config` and `.load_config`, or mention the "raw" `.config` anywhere above? It seems to me we should.